src/nora/components/NoraButtonInput/NoraButtonInput.module.scss

```diff
@@ -0,0 +1,41 @@
+.NoraButtonInputContainer {
+  display: flex;
+}
+
+.Input {
+  flex: 1;
+}
+
+.InputRight {
+  composes: Input;
+  > input {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+}
+
+.InputLeft {
+  composes: Input;
+  > input {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+}
+
+.Button {
+  width: var(--Space-40);
+  background: transparent;
+  color: var(--NoraNeutral-300);
+}
+.ButtonLeft {
+  composes: Button;
+  margin-right: -1px;
+}
+.ButtonRight {
+  composes: Button;
+  margin-left: -1px;
+}
+
+.Disabled {
```

Comment:
Per @sriram-ethos other comment regarding hover, I agree that it's missing here. I think we should do some sort of `:not(.Disabled):hover` or even negate the hover rules in .Disabled itself (might be more straight forward).

Reply:
Actually, I think since we do `pointer-events: none` and `.Disabled` is removing button interactively, all above is probably not even needed. Put differently, my guess is we just add the hover functionality, and when the button's disabled, it will "just work" and no hover rules will be able to get applied.

Reply:
I agree.  I think its better to start from scratch (similar to other props) as the behavior is not the same as the TextInput.